### PR TITLE
solves GH 43 - Use SiteURL for all calls from the webapp

### PR DESCRIPTION
**Pull request by maisnamrajusingh**
#### Summary
Use SiteURL for all calls from the webapp
#### Ticket Link
https://github.com/mattermost/mattermost-plugin-webex/issues/43

### CODE REVIEW

File: webapp/src/index.js [hunk 1]
```diff
@@ -20,7 +23,23 @@ class Plugin {
...
+    if (config && config.SiteURL) {
+        basePath = new URL(config.SiteURL).pathname;
+
+        if (basePath && basePath[basePath.length - 1] === '/') {
+            basePath = basePath.substr(0, basePath.length - 1);
+        }
+    }
+
+    return basePath;
+};
```
Comment (hanzei) [inline webapp/src/index.js:45]: Nit pick: Add linebreak

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Comment by jasonblais** [comment]
@hanzei should @cpoile still be the second reviewer, or a second member of the integrations team?

**Review by cpoile — APPROVED** [review]
Nice @maisnamrajusingh. Can you add the linebreak @hanzei mentioned, then we can merge this. Thanks!

**Comment by jasonblais** [comment]
@maisnamrajusingh is this ready for final review?

**Comment by maisnamrajusingh** [comment]
@hanzei @jasonblais ready now. sorry for the delay. I thought I had pushed this already.

**Review by hanzei — APPROVED** [review]

**Review by hanzei — APPROVED** [review]

**Review by hanzei — APPROVED** [review]

**Pull request merged**